src/role_storagefiller.js

```diff
@@ -16,6 +16,7 @@ roles.storagefiller.killPrevious = true;
 roles.storagefiller.settings = {
   layoutString: 'MC',
   amount: [1, 4],
+  maxLayoutAmount: 1
```

Comment:
The creep will have 4 carry parts, so 200 carryCapacity, that should be enough the clean the link before the next transfer happens